src/main/java/frc/robot/hardware/servo/Servo.java

```diff
@@ -0,0 +1,86 @@
+package frc.robot.hardware.servo;
+
+/**
+ * A wrapper class around Servo.
+ * @see edu.wpi.first.wpilibj.Servo
+ */
+public class Servo {
+    private final edu.wpi.first.wpilibj.Servo servo;
+    private boolean inverted = false;
+
+    /**
+     * @param port the PWM port the servo is connected to
+     */
+    public Servo(int port) {
+        servo = new edu.wpi.first.wpilibj.Servo(port);
+    }
+
+    /**
+     * Sets the speed of the servo.
+     * If the servo is inverted, the speed will be set to 1 - speed.
+     * @param speed the speed to set the servo to
+     */
+    public void set(double speed) {
+        if (inverted) {
+            servo.set(1 - speed);
+        } else {
+            servo.set(speed);
+        }
+    }
+
+    /**
+     * Sets the angle of the servo.
+     * @param angle the angle to set the servo to
+     * @see edu.wpi.first.wpilibj.Servo#setAngle(double)
+     */
+    public void setAngle(double angle) {
+        servo.setAngle(angle);
```

Comment:
I am considering removing the inverted state all together, as I'm not sure if it should be considered when using servos. I only implemented it because the Motor classes have it, and I wanted to keep it fairly similar to them.